pl_PL/LC_MESSAGES/trophies.po

```diff
@@ -3,123 +3,123 @@
+msgstr "Był/a na imprezie urodzinowej z okazji piątych urodzin Sudomemo"
 
 msgid "TROPHY_NAME_legacy_citizen"
-msgstr "Legacy Citizen"
+msgstr "Były Citizen"
 
 msgid "TROPHY_DESC_legacy_citizen"
-msgstr "Had Sudomemo Citizenship before it became Plus"
+msgstr "Miał/a Sudomemo Citizenship zanim zmieniło się na Sudomemo PLus"
 
 msgid "TROPHY_NAME_meet_lazy_paisley"
-msgstr "Meet Lazy Paisley"
+msgstr "Spotkaj Lazy Paisley"
 
 msgid "TROPHY_DESC_meet_lazy_paisley"
-msgstr "Enjoys his simple life. And food."
+msgstr "Lubi swoje proste życie. I jedzenie."
 
 msgid "TROPHY_NAME_meet_bailey_flitwing"
-msgstr "Meet Bailey Flitwing"
+msgstr "Spotkaj Bailey Flitwing"
 
 msgid "TROPHY_DESC_meet_bailey_flitwing"
-msgstr "Curious about the world outside her nest."
+msgstr "Ciekawa świata poza jej gniazdem."
 
 msgid "TROPHY_NAME_meet_ziggy_starburst"
-msgstr "Meet Ziggy Starburst"
+msgstr "Spotkaj Ziggy Starburst"
 
 msgid "TROPHY_DESC_meet_ziggy_starburst"
-msgstr "Living his life day by day with a touch of added flair."
+msgstr "Żyje skromnie, lecz z klasą."
 
 msgid "TROPHY_NAME_meet_russell_ramblewood"
-msgstr "Meet Russell Ramblewood"
+msgstr "Spotkaj Russell Ramblewood"
 
 msgid "TROPHY_DESC_meet_russell_ramblewood"
-msgstr "He's flamboyant to a fault, but still wants to hear about your day."
+msgstr "Pozornie może się wydawać dość niemiły, ale zawsze przejmuje się jak minął twój dzień."
 
 msgid "TROPHY_NAME_meet_mitsuo_meow"
-msgstr "Meet Mitsuo Meow"
+msgstr "Spotkaj Mitsuo Meow"
 
 msgid "TROPHY_DESC_meet_mitsuo_meow"
-msgstr "Wants to bring good luck and happiness to everyone around her."
+msgstr "Chce przynieść szczęście i pomyślność dla wszystkich dookoła niej."
 
 msgid "TROPHY_NAME_neighborhood_watch"
-msgstr "Neighborhood Watch"
+msgstr "Czujny patrolowiec"
 
 msgid "TROPHY_DESC_neighborhood_watch"
-msgstr "Correctly flag content that violates our Content Policy"
+msgstr "Prawidłowo zgłoś naruszenie zasad dot. twórczości Sudomemo"
 
 msgid "TROPHY_NAME_one_million_flips_event"
-msgstr "One Million Flipnotes Event"
+msgstr "Impreza z okazji miliona Flipnotes"
 
 msgid "TROPHY_DESC_one_million_flips_event"
-msgstr "Attend the One-Million Flipnotes event"
+msgstr "Był/a na imprezie z okazji miliona Flipnotes"
 
 msgid "TROPHY_NAME_translator"
-msgstr "Translator"
+msgstr "Tłumacz"
 
 msgid "TROPHY_DESC_translator"
-msgstr "Contribute translations to make Sudomemo open to everyone"
+msgstr "Pomóż w tłumaczeniu Sudomemo"
 
 msgid "TROPHY_NAME_weekly_topic_winner"
-msgstr "Weekly Topic Winner"
+msgstr "Zwycięzca Tematu Tygodnia"
 
 msgid "TROPHY_DESC_weekly_topic_winner"
-msgstr "Submit a winning entry to Sudomemo's Weekly Topic"
+msgstr "Twój Flipnote wygrał Temat Tygodnia"
 
 msgid "TROPHY_NAME_folder"
 msgstr "Folder"
 
 msgid "TROPHY_DESC_folder"
-msgstr "Donate spare compute time to Folding@Home to help fight disease"
+msgstr "Wesprzyj nasz zespół na Folding@Home by pomóc w walce z chorobami"
```

Comment:
When translated back, this says "Support our team on Folding@Home". Is this a good way to put this?

Reply:
The focus should be purely on donating your computer's power and fighting disease.